common/Server/eos/core/eos_core.sqf

```diff
@@ -107,14 +119,21 @@ sleep 0.25;
 	for "_counter" from 1 to _dGrps do {
 	if (isnil "_dGrp") then {_dGrp=[];};
 
-		_newpos=[_mkr,50] call EOS_fnc_findSafePos;
+		_newpos=[_mkr,100] call EOS_fnc_findSafePos;
```

Comment:
performance could be an issue gotta keep an eye on it. But for now best do the light vehicles too